samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -65,6 +65,11 @@ static const char cache_dir_message[] = "Optional. Enables caching of loaded mod
 static const char load_from_file_message[] = "Optional. Loads model from file directly without read_model."
                                              " All CNNNetwork options (like re-shape) will be ignored";
 
+/// @brief message for run frequency
+static const char run_frequency_message[] =
```

Comment:
in serving it's typically named "requests rate"

Reply:
Name changed. Please check if the new naming matches the OV style